script/odom_map_transform_publisher.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+import rospy
+import numpy as np
+from geometry_msgs.msg import TransformStamped, Quaternion, Transform
+from tf2_msgs.msg import TFMessage
+from iai_naive_kinematics_sim.srv import SetMapOdomTransform, SetOdomMapTransformRequest, SetOdomMapTransformResponse 
```

Comment:
`SetOdomMapTransformRequest` not found. It exists with `MapOdom`, but you don't use it anyway, so just remove the imports.